contributor/outline.md

```diff
@@ -0,0 +1,46 @@
+# InnerSource Contributor Content Outline
+
+## Introduction/Rationale
+   - Quick Overview of Roles
+      - Explain how contributor role relates to other roles
+   - Discuss the value of contribution
+      - ‘With many eyes, all bugs are shallow’
+      - Improve cross-company collaboration
+      - Reduce single points of failure (team or personal)
+   - Contribution Rationale
+      - Prevent/reduce duplication of efforts through many teams
+      - Increasing personal reputation
+      - Things that you gain as a contributor.
+         - Help from others on the project in implementing your feature.
+         - Help from the broader community in maintaining what is written.  You are not stuck with the long-term burden of maintenance.
+         - Heavy influence into the final product.
+         - Assurance that the final product will be useful for you.
+
+## Mental/Attitude/Communications
+   - A lot of times I want to make a contribution but it’s hard to be OK with them saying “no” or that their goals are different then mine as a contributor.  How do you walk away from that feeling good.
+      - Adjust and have realistic expectations
+      - Empathy and humility
+   - Have a contributing mindset - think outside your project and look for places to contribute instead of reinventing the wheel.
+      - A contributing mindset starts with a shared use mindset.  You’ll likely have to be comfortable using other folks’ stuff before it is realistic to contribute to it.
+         - Think about contributing.  Every time you’re working on something in a project that is not related to the purpose or domain of that project, look for the proper home for that code.
+   - Use the Guest in a home analogy.
+   - Nothing too small to contribute!!!!!!
+   - Understand that you have to ‘sell’ your idea - communicate early and often -before- investing a lot of time in coding an idea (fail fast)
+   - ‘Selling’ your participation to your management team
+   - Understand the general purpose of the project enough to know what your contribution makes sense as an overall part of the project instead of just your personal snowflake
+
+## Technical Concerns
```

Comment:
I'm not sure what separates this advice from the other sections.  How are these things technical?

Perhaps rename this section to something like
> Working with code

Reply:
Agreed that it might not be the best place for them.  But, per your earlier point, I don't think we want to have a 'working with code' section if we want to allow for contributions that are broad (docs, bug reports, questions answered, etc.).